{
  "NServiceBus.MessageId": "4c2f1d72-d093-4e18-8b9e-a9cf011ff2f6",
  "NServiceBus.MessageIntent": "Send",
  "NServiceBus.ConversationId": "419307b9-b77b-4998-90dd-a9cf011ff2fa",
  "NServiceBus.CorrelationId": "4c2f1d72-d093-4e18-8b9e-a9cf011ff2f6",
  "NServiceBus.OriginatingMachine": "DESKTOP-KBE494B",
  "NServiceBus.OriginatingEndpoint": "WEB",
  "$.diagnostics.originating.hostid": "bbea3ddd08940f60852f47356a2cb9a8",
  "NServiceBus.ReplyToAddress": "WEB-WEB",
  "NServiceBus.ContentType": "text\/xml",
  "NServiceBus.EnclosedMessageTypes": "LimeTest.Messages.Poems.GetPoems, LimeTest.Messages, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
  "NServiceBus.Version": "7.1.6",
  "NServiceBus.TimeSent": "2019-01-08 17:28:23:475286 Z",
  "NServiceBus.ExceptionInfo.ExceptionType": "System.InvalidOperationException",
  "NServiceBus.Retries.Timestamp": "2019-01-08 17:28:56:123846 Z",
  "NServiceBus.ExceptionInfo.HelpLink": null,
  "NServiceBus.ExceptionInfo.Message": "No handlers could be found for message type: LimeTest.Messages.Poems.GetPoems",
  "NServiceBus.ExceptionInfo.Source": "NServiceBus.Core",
  "NServiceBus.ExceptionInfo.StackTrace": "System.InvalidOperationException: No handlers could be found for message type: LimeTest.Messages.Poems.GetPoems\u000d\u000a   в NServiceBus.LoadHandlersConnector.<Invoke>d__1.MoveNext()\u000d\u000a--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   в NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext()\u000d\u000a--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   в NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext()\u000d\u000a--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   в NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext()\u000d\u000a--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   в NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext()\u000d\u000a--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   в System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\u000d\u000a   в NServiceBus.LearningTransportMessagePump.<ProcessFile>d__10.MoveNext()",
  "NServiceBus.TimeOfFailure": "2019-01-08 17:29:27:418061 Z",
  "NServiceBus.FailedQ": "Report",
  "NServiceBus.ProcessingMachine": "DESKTOP-KBE494B",
  "NServiceBus.ProcessingEndpoint": "Report",
  "$.diagnostics.hostid": "81ce4ed5c9f23073b2e439b2cac668f7",
  "$.diagnostics.hostdisplayname": "DESKTOP-KBE494B"
}